.github/workflows/ci.yml

```diff
@@ -10,18 +10,6 @@ jobs:
         otp_vsn: ['24', '25', '26']
         rebar3_vsn: ['3.22']
     steps:
-      - name: Install Git 2.18
-        shell: bash
-        run: |
-          sudo apt-get update
-          sudo apt-get -y install tcl tcl-dev gettext libcurl4-openssl-dev
-          cd /usr/src/
-          sudo wget https://github.com/git/git/archive/v2.18.0.tar.gz -O git.tar.gz
-          sudo tar -xf git.tar.gz
-          cd git-*
-          sudo make prefix=/usr/local all
-          sudo make prefix=/usr/local install
-          cd $HOME
```

Comment:
Can't remember why I added this in the past (commit message doesn't explain it) but I did, and now I remove it because it takes a looooong time and seems unnecessary.